cmd/src/snapshot_databases.go

```diff
@@ -19,13 +19,13 @@ func init() {
 Note that these commands are intended for use as reference - you may need to adjust the commands for your deployment.
 
 USAGE
-	src [-v] snapshot databases [--targets=<docker|k8s|"targets.yaml">] [--run] <pg_dump|docker|kubectl> 
+	src [-v] snapshot databases [--targets=<docker|k8s|"targets.yaml">] [--run] <pg_dump|docker|kubectl>
 
 TARGETS FILES
 	Predefined targets are available based on default Sourcegraph configurations ('docker', 'k8s').
 	Custom targets configuration can be provided in YAML format with '--targets=target.yaml', e.g.
 
-		primary:
```

Comment:
I don't think we should rename this in this change, as it's unrelated to the main thing being fixed, and is also a configuration change

I also don't think this is correct? If anything it should be called `frontend` https://sourcegraph.sourcegraph.com/github.com/sourcegraph/sourcegraph/-/blob/migrations/README.md but either way, I dont think we should change that here